util/new_keyboard.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import argparse
+from distutils.dir_util import copy_tree
+import fileinput
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+def print_error(message):
+    print("[\033[0;91mERROR\033[m] " + message, file=sys.stderr)
+
+def prompt(message, default=""):
+    """
+    Asks the user for input, showing the default value, if any, in brackets.
```

Comment:
```suggestion
    """Ask the user for input, showing the default value in brackets."""
```
PEP 257 [recommends](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) having `"""No newlines in one-line docstrings."""`. Also:

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

The "if any" clarification isn't necessary, imo.